extract.py

```diff
@@ -0,0 +1,81 @@
+"""Script that connects to 'data-eng-plants-api'
+   Extracts the data
+   Puts it into a Pandas DataFrame"""
+import logging
+from multiprocessing import Pool, cpu_count
+import requests
+import pandas as pd
+
+
+URL = "https://data-eng-plants-api.herokuapp.com/plants/"
+TOTAL_NUMBER_OF_PLANTS = 50
+
+
+def get_plant_data(plant_id: int) -> dict:
+    """GET request to collect plant data from API."""
+    try:
+        response = requests.get(URL + str(plant_id), timeout=1000)
```

Comment:
does this mean it will only timeout after 1000 seconds? is that too long given there's new data every minute?